pkg/kubectl/cmd/exec/exec.go

```diff
@@ -40,27 +44,37 @@ import (
 
 var (
 	execExample = templates.Examples(i18n.T(`
-		# Get output from running 'date' from pod 123456-7890, using the first container by default
-		kubectl exec 123456-7890 date
+		# Get output from running 'date' command from pod mypod,
+		# using the first container by default
```

Comment:
Good change :+1: but please the formatting (iow. the description in a single line) like before. I don't want to end up with too long help like in `kubectl run` case.

Reply:
updated